lib/assets/javascripts/cable.coffee

```diff
@@ -0,0 +1,19 @@
+#= require_self
+#= require cable/consumer
+
+@Cable =
+  PING_IDENTIFIER: "_ping"
+
+  createConsumer: (url) ->
+    new Cable.Consumer websocketUrl(url)
+
+  websocketUrl = (url) ->
+    if url and not /^wss?:/i.test(url)
+      a =  document.createElement('a')
+      a.href = url
+      # ie fix for relative urls
+      a.href = a.href
+      a.protocol = a.protocol.replace("http", "ws")
+      a.href
+    else
+      url
```

Comment:
This file was renamed to `cable.coffee.erb` in master and has been modified since. You'll need to move your changes over.